### ISSUE TITLE
There is error when we try to build type script project that use your package for checking ADA balance.

### ISSUE THREAD

**Issue by emadpanah** [blockfrost/openapi#312]
node_modules/@blockfrost/blockfrost-js/node_modules/@blockfrost/openapi/lib/functions/schema.d.ts:7:13 - error TS2315: Type 'ErrorObject' is not generic.

7     errors: import("ajv").ErrorObject<string, Record<string, any>, unknown>[]                                      | null | undefined;
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@blockfrost/blockfrost-js/node_modules/@blockfrost                                  /openapi/lib/functions/schema.d.ts:7

**Comment by slowbackspace** [comment]
Hi @emadpanah, we've just published a new version [0.1.59](https://www.npmjs.com/package/@blockfrost/openapi/v/0.1.59) that should fix the problem.

**Comment by slowbackspace** [comment]
Hey @emadpanah I'll close the issue as the problem should be solved. If it still doesn't work for you then let us know.
